cartoframes/viz/source.py

```diff
@@ -205,9 +205,9 @@ def _get_credentials(self):
         context = self._get_context()
```

Comment:
`Source` should receive `credentials` instead of `context`

Reply:
It is matter of the following PR